packages/auth/amplify_auth_cognito_test/test/state/credential_store_state_machine_test.dart

```diff
@@ -38,16 +38,16 @@ void main() {
     // Load an empty credential store.
     test('loadCredentialStore (empty)', () async {
       stateMachine.dispatch(
-        const CredentialStoreEvent.migrateLegacyCredentialStore(),
+        const CredentialStoreEvent.loadCredentialStore(),
```

Comment:
I think I missed the comment(s) in first PR that is the sets the context for changes in this PR. Do you mind linking that or summarizing why this is changed?

Reply:
https://github.com/aws-amplify/amplify-flutter/pull/2482#discussion_r1095075656

The event is not really necessary and so this helps trim down the API and makes `manager.loadCredentials` single-purpose. Basically, it's easier to do the right thing always.

Reply:
thanks